paragon/_overrides.scss

```diff
@@ -1,20 +1,1115 @@
 // Use this file to define style overrides for @edx/paragon
 // This file is included after all Paragon styles, but should generally avoid using private mixins in Paragon.
 
-@import "~@openedx/paragon/styles/css/themes/light/variables";
-@import "~@openedx/paragon/styles/css/themes/light/utility-classes";
+@import "@openedx/paragon/styles/css/themes/light/variables";
+@import "@openedx/paragon/styles/css/themes/light/utility-classes";
+@import "@openedx/paragon/styles/scss/core/core";
 
-@import url('https://cdn.jsdelivr.net/npm/@openedx/paragon@23.0.0-alpha.1/styles/css/themes/light/utility-classes.min.css');
-@import url('https://cdn.jsdelivr.net/npm/@openedx/paragon@23.0.0-alpha.1/dist/core.min.css');
+@import url("https://cdn.jsdelivr.net/npm/@openedx/paragon@23.0.0-alpha.1/styles/css/themes/light/utility-classes.min.css");
+@import url("https://cdn.jsdelivr.net/npm/@openedx/paragon@23.0.0-alpha.1/dist/core.min.css");
 
 @font-face {
-  font-family: 'DIN Next LT Arabic';
-  src: url('./fonts/din-next-lt-arabic-bold.ttf');
+  font-family: "DIN Next LT Arabic";
```

Comment:
DIN is no longer used, please use both IBM Plex and Inter as shown in the variables section.